src/Cli/dotnet/Commands/Reference/Remove/ReferenceRemoveCommand.cs

```diff
@@ -5,7 +5,7 @@
 
 using System.CommandLine;
 using Microsoft.Build.Evaluation;
```

Comment:
adding a comment over where for whether you were going to add a test for https://github.com/dotnet/sdk/pull/49314/files